sklego/meta/estimator_transformer.py

```diff
@@ -17,15 +16,18 @@ class EstimatorTransformer(TransformerMixin, MetaEstimatorMixin, BaseEstimator):
 
     :param estimator: An instance of the estimator that should be used for the transformation
     :param predict_func: The function called on the estimator when transforming e.g. (`predict`, `predict_proba`)
+    :param check_X: Whether to check the input data for NaNs, Infs and non-numeric values
     """
 
-    def __init__(self, estimator, predict_func="predict"):
+    def __init__(self, estimator, predict_func="predict", check_X=True):
```

Comment:
[This discussion](https://github.com/koaning/scikit-lego/pull/580#discussion_r1356711192) may be relevant. Once we agree on a direction this PR can follow the same path.

Reply:
sounds good. thanks @koaning

Reply:
done!